pvlib/test/test_solarposition.py

```diff
@@ -754,8 +754,125 @@ def test_sun_rise_set_transit_geometric(expected_rise_set_spa, golden_mst):
                        atol=np.abs(expected_transit_error).max())
 
 
-# put numba tests at end of file to minimize reloading
+def test_spencer_mc():
```

Comment:
Please follow this test pattern so that we can be sure the output is correct to within some tolerance: https://github.com/pvlib/pvlib-python/blob/d68617c03a2bd6cc95100d91d8de5f1069ae13cc/pvlib/test/test_solarposition.py#L420-L455

Reply:
Good Idea. I will bring it in the next commit